LDAR_Sim/src/virtual_world/sites.py

```diff
@@ -331,11 +331,11 @@ def get_required_surveys(self, method_name) -> int:
     def do_site_deployment(self, method_name) -> bool:
         return self._deploy_method[method_name]
 
-    def get_method_survey_time(self, method_name) -> float:
+    def get_method_survey_time(self, method_name) -> int:
         survey_time: float = 0
         for eqg in self._equipment_groups:
             survey_time += eqg.get_survey_time(method_name=method_name)
-        return survey_time
+        return round(survey_time)
```

Comment:
Nit: I think this should be ceiling instead of round, as we're doing with time, and if you have a fraction of a minute, I would use the next integer. It's more conservative but I think better practice. 
An example of where this might be relevant is if an equipment took 0.4 minutes to survey. We would want it to say 1 minute to survey.

Reply:
I disagree here. This is the sum of all equipment groups survey time, so an equipment having a survey time of 0.4 minutes will not necessary mean it does not contribute to the site survey time. Additionally, practically, survey times must be integers. If survey time is provided at the equipment group level this will not matter. The purpose is to deal with the situation in which site survey time is evenly split among equipment groups and we want to go back to the original site survey time. In this case we expect to always be close to an integer number ie: 30.00000000003 or 29.9999999997 and so taking the ceiling here would be incorrect.